src/transformStateWithClones.js

```diff
@@ -64,8 +64,35 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const statesArr = [];
+  let stateCopObj = Object.assign({}, state);
+
+  transforms.forEach((operationObj) => {
```

Comment:
```suggestion
  transforms.forEach(({ properties, operation }) => {
```
Можно использовать properties operation вместо operationObj.properties operationObj.operation.